orchagent/fdborch.cpp

```diff
@@ -179,6 +241,59 @@ void FdbOrch::doTask(Consumer& consumer)
     }
 }
 
+void FdbOrch::doTask(NotificationConsumer& consumer)
+{
+    SWSS_LOG_ENTER();
+
+    if (!gPortsOrch->isInitDone())
+    {
+        return;
+    }
+
+    std::string op;
+    std::string data;
+    std::vector<swss::FieldValueTuple> values;
+
+    consumer.pop(op, data, values);
+
+    if (op == "ALL")
+    {
+        /*
+         * so far only support flush all the FDB entris
+         * flush per port and flush per vlan will be added later.
+         */
+        sai_status_t status;
+        sai_attribute_t attr;
+        attr.id = SAI_FDB_FLUSH_ATTR_ENTRY_TYPE;
+        attr.value.s32 = SAI_FDB_FLUSH_ENTRY_TYPE_DYNAMIC;
+
+        status = sai_fdb_api->flush_fdb_entries(gSwitchId, 1, &attr);
```

Comment:
when flushing all, there should be 0 attributes passed, all static and dynamic entries should be cleared

Reply:
revised.